src/Compilers/CSharp/Test/Symbol/Symbols/UnsignedRightShiftTests.cs

```diff
@@ -0,0 +1,1873 @@
+        [InlineData("double", "string")]
+        [InlineData("double", "bool")]
+        [InlineData("double", "char")]
+        [InlineData("double", "sbyte")]
+        [InlineData("double", "short")]
+        [InlineData("double", "int")]
+        [InlineData("double", "long")]
+        [InlineData("double", "byte")]
+        [InlineData("double", "ushort")]
+        [InlineData("double", "uint")]
+        [InlineData("double", "ulong")]
+        [InlineData("double", "nint")]
+        [InlineData("double", "nuint")]
+        [InlineData("double", "float")]
+        [InlineData("double", "double")]
+        [InlineData("double", "decimal")]
+        [InlineData("decimal", "object")]
+        [InlineData("decimal", "string")]
+        [InlineData("decimal", "bool")]
+        [InlineData("decimal", "char")]
+        [InlineData("decimal", "sbyte")]
+        [InlineData("decimal", "short")]
+        [InlineData("decimal", "int")]
+        [InlineData("decimal", "long")]
+        [InlineData("decimal", "byte")]
+        [InlineData("decimal", "ushort")]
+        [InlineData("decimal", "uint")]
+        [InlineData("decimal", "ulong")]
+        [InlineData("decimal", "nint")]
+        [InlineData("decimal", "nuint")]
+        [InlineData("decimal", "float")]
+        [InlineData("decimal", "double")]
+        [InlineData("decimal", "decimal")]
+        public void BuiltIn_CompoundAssignment_02(string left, string right)
+        {
+            var source1 =
+@"
+class C
+{
+    static void Main()
+    {
+        " + left + @" x = default;
+        " + right + @" y = default;
+        x >>= y;
+        x >>>= y;
+    }
+}
+";
+            var compilation1 = CreateCompilation(source1, options: TestOptions.DebugDll,
+                                                 parseOptions: TestOptions.RegularPreview);
+            compilation1.VerifyEmitDiagnostics(
+                // (8,9): error CS0019: Operator '>>=' cannot be applied to operands of type 'double' and 'char'
+                //         x >>= y;
+                Diagnostic(ErrorCode.ERR_BadBinaryOps, "x >>= y").WithArguments(">>=", left, right).WithLocation(8, 9),
+                // (9,9): error CS0019: Operator '>>>=' cannot be applied to operands of type 'double' and 'char'
+                //         x >>>= y;
+                Diagnostic(ErrorCode.ERR_BadBinaryOps, "x >>>= y").WithArguments(">>>=", left, right).WithLocation(9, 9)
+                );
+        }
+
+        [Fact]
+        public void CollectionInitializerElement()
+        {
+            var source1 =
+@"
+class C
+{
+    static void Main()
+    {
+        var x = int.MinValue;
+        var y = new System.Collections.Generic.List<int>() {
+            x >>= 1,
+            x >>>= 1 
+            };
+    }
+}
+";
+            var compilation1 = CreateCompilation(source1, options: TestOptions.DebugDll,
+                                                 parseOptions: TestOptions.RegularPreview);
```

Comment:
nit: all these tests don't need `TestOptions.RegularPreview` (that's the default)